lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -226,6 +250,15 @@ class RenderPopover extends RenderShiftedBox {
     }
   }
 
+  bool get extendAndClipContentOverArrow => _extendAndClipContentOverArrow;
+  bool _extendAndClipContentOverArrow;
+  set extendAndClipContentOverArrow(bool value) {
+    if (value != _extendAndClipContentOverArrow) {
```

Comment:
As a matter of consistency, render object property setters should use a guard statement instead of an all-encompassing if-statement:

```dart
if (newValue == _extendAndClipContentOverArrow) {
  return;
}

_extendAndClipContentOverArrow = newValue;
markNeedsLayout();
```